modules/demo/rapids-api-server/routes/graphology/index.js

```diff
@@ -0,0 +1,146 @@
+      let message = 'Unknown error';
+      let result  = {'params': JSON.stringify(request.query), success: false, message: message};
+
+      console.log(result);
+      if (request.query.filename.search('http') != -1) {
+        message = 'Remote files not supported yet.'
+        console.log(result);
+      } else {
+        message = 'File is not remote';
+        console.log(message);
+        // does the file exist?
+        const path = Path.join(__dirname, request.query.filename);
+        console.log('Does the file exist?');
+        try {
+          const stats = await Stat(path);
+          if (stats == null) {
+            message = 'File does not exist at ' + path;
+            console.log(message);
+            result.message = message;
+            reply.code(200).send(result);
+          } else {
+            // did the file read?
+            message = 'File is available';
+            console.log(message);
+            result.success   = true;
+            message          = 'Successfully parsed json file onto GPU.';
+            result.message   = message;
+            const graphology = gpu_cache.readGraphology(path);
+            gpu_cache.setDataframe('nodes', graphology['nodes']);
+            gpu_cache.setDataframe('edges', graphology['edges']);
+            gpu_cache.setDataframe('clusters', graphology['clusters']);
+            gpu_cache.setDataframe('tags', graphology['tags']);
+            reply.code(200).send(result);
+          }
+        } catch {
+          message        = 'Exception reading file.';
+          result.message = message;
+          reply.code(200).send(result);
+        };
+      }
+    }
+  });
+
+  fastify.route({
+    method: 'GET',
+    url: '/get_column/:table/:column',
+    schema: {querystring: {table: {type: 'string'}, 'column': {type: 'string'}}},
+    handler: (request, reply) => {
+      let message = 'Not Implemented';
+      let result  = {'params': JSON.stringify(request.params), success: false, message: message};
+      console.log(request.params.table);
+      const table = gpu_cache.getDataframe(request.params.table);
+      console.log(table);
+      if (table == undefined) {
+        result.message = 'Table not found';
+        reply.code(404).send(result);
+      } else {
+        const column = table.get(request.params.column);
+        console.log(column);
+        console.log(result);
+        console.log('reply.stream()');
+        console.log('reply.streamed');
+        console.log(column.toArrow());
+        console.log(column.toArrow().values.length);
+        // reply.code(200).send(column.toArray());
+        // reply.header('Content-Type', 'Application/octet-stream');
+        // reply.header('Accepts', 'Application/octet-stream');
+        // reply.code(200);
+        // reply.header('Content-Length', column.toArrow().values.length);
+        RecordBatchStreamWriter.writeAll(column.toArrow()).pipe(reply.stream({objectMode: false}));
```

Comment:
Hey @trxcllnt , do you mind commenting as to why this may not be working? It returns an empty response instead of the arrow data. I'm following the patterns at `fastify-arrow`. What gives, do you think? Thanks!

Reply:
@thomcom the Arrow IPC writer accepts a Table as input, it can't write individual columns.

Reply:
:heart: